easybuild/easyconfigs/l/LAMMPS/LAMMPS-patch_4Feb2020-foss-2019a-Python-3.7.2-CUDA-10.1.105-kokkos.eb

```diff
@@ -0,0 +1,121 @@
+name = 'LAMMPS'
+version = 'patch_4Feb2020'
+local_cuda_ver = '10.1.105'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-CUDA-%s' % local_cuda_ver
+versionsuffix += '-kokkos'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True, 'usempi': True, 'cstd': 'c++11'}
+
+# 'https://github.com/lammps/lammps/archive/'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['3bf3de546ede34ffcd89f1fca5fd66aa78c662e7c8a76e30ce593e44a00d23ce']
+
+local_source_dir_name = '%(namelower)s-%(version)s'
+
+builddependencies = [
+    ('CMake', '3.13.3'),
+    ('pkg-config', '0.29.2'),
+    ('archspec', '0.1.0', '', True),
```

Comment:
@Darkless012 We shouldn't rely on an `archspec` that was installed for the system Python here, since that may be a different Python version.

We should use an easyconfig file for `archspec` on top of the same `Python` you're using as a dep here (3.7.x); see #9898